From 72f1a41e520e326fdf25c8c92e7165ed8fb82255 Mon Sep 17 00:00:00 2001
From: Raghavendra Talur <rtalur@redhat.com>
Date: Wed, 30 Dec 2015 16:19:44 +0530
Subject: [PATCH 246/251] glusterd: GD_OP_VERSION should not be a released one

        Backport of http://review.gluster.org/13117

performance.resync-failed-syncs-after-fsync was
introduced after 3.7.6 was released. Hence it should
use 3_7_7 as op version not 3_7_6.

Change-Id: If4def1bf0fdc9fa4938ccb78308bec77eeaa2284
BUG: 1219794
Reviewed-on: http://review.gluster.org/13117
Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
Tested-by: Gluster Build System <jenkins@build.gluster.com>
Signed-off-by: Raghavendra Talur <rtalur@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/65752
Reviewed-by: Raghavendra Gowdappa <rgowdapp@redhat.com>
Tested-by: Raghavendra Gowdappa <rgowdapp@redhat.com>
---
 xlators/mgmt/glusterd/src/glusterd-volume-set.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/xlators/mgmt/glusterd/src/glusterd-volume-set.c b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
index da1f15a..44f74a3 100644
--- a/xlators/mgmt/glusterd/src/glusterd-volume-set.c
+++ b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
@@ -1234,7 +1234,7 @@ struct volopt_map_entry glusterd_volopt_map[] = {
         { .key        = "performance.resync-failed-syncs-after-fsync",
           .voltype    = "performance/write-behind",
           .option     = "resync-failed-syncs-after-fsync",
-          .op_version = GD_OP_VERSION_3_7_6,
+          .op_version = GD_OP_VERSION_3_7_7,
           .flags      = OPT_FLAG_CLIENT_OPT,
           .description = "If sync of \"cached-writes issued before fsync\" "
                          "(to backend) fails, this option configures whether "
-- 
1.7.1

